Automattic/newspack-custom-content-migrator
# Test phpcs multiple files

## Description
In https://github.com/Automattic/newspack-custom-content-migrator/pull/526 I didn't think to check what would happen if mulitple files had changes. Turns out that breaks, so this is to fix that. 

## How to test
* Push a broken php file and expect failure from the run
* Fix the file and expect the run to succed.

I tested this, and I'll merge so I'm not in the way in other PRs.
